### PR TITLE
config eMapis css

### PR DESCRIPTION
**Pull request by pfanguin**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,11 +6,15 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>eMapis</title>
     <link href="lib/bootstrap.min.css" rel="stylesheet">
+    <link href="//netdna.bootstrapcdn.com/bootstrap/3.0.0/css/bootstrap.min.css" rel="stylesheet">
```
Comment (oterral) [inline index.html]: no need for this bootstrap css are loaded just one line before

File: less/app.less [hunk 2]
```diff
@@ -1,33 +1,65 @@
 @import "map.less";
 @import "search.less";
+
+html {
+  overflow:hidden
```
Comment (oterral) [inline less/app.less]: space after the `:` and the `;`at the end

File: less/app.less [hunk 3]
```diff
@@ -39,5 +71,105 @@
   background-color: white;
 }
 
+#map .ol-zoom {
+  position: absolute;
```
Comment (oterral) [inline less/app.less]: move to map.less

File: less/app.less [hunk 4]
```diff
@@ -39,5 +71,105 @@
...
+}
+
+#map .ol-zoom-out {
+  width: 44px;
+  height: 44px;
+  background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACwAAAAsCAMAAAApWqozAAAARVBMVEUAAAAzMzMzMzMzMzMzMzMzMzMzMzMzMzMzMzMzMzMzMzMzMzMzMzMzMzP////MzMxZWVny8vLj4+OmpqaMjIyAgIBNTU2nJLiyAAAADXRSTlMAzyAwEEDvv6+PYFDfGXMyHwAAAPtJREFUOMullW0PgyAMhFdx06kF3///T13CgMKsZhefj3oh1yu0j/sY85es6lpiD7VddSntGy5o+lPpk/gAPVVp/WaVd61oiU+gg3Xz4lNe5icE0WrqSvegQ7nvorZxXZxb1rGoMsuMhW22gXljQRIUE7uzGW4XI6lv4mCyBZN4ib1s0rlRK+p0dhNi44izBxxHvvF1qTarkKrsvLjlwKyJZw60XjzE6qzKWOTBgVUXrxwoxIv/xxn+w5KLjWShiSUPA4kRG3iBBEQHNAVoN3SRkCsKXX7kWUEPFhkF0JABxleND0ahoisP+DDH1wS+gGS1DeerTTdkHrf5AFOHQ1ckgnmJAAAAAElFTkSuQmCC');
+}
+
+#map .ol-zoom-in:hover,
+#map .ol-zoom-out:hover {
```
Comment (oterral) [inline less/app.less]: move to map.less

File: less/app.less [hunk 5]
```diff
@@ -39,5 +71,105 @@
...
+
+#map .ol-zoom-in {
+  width: 44px;
+  height: 44px;
+  margin-top: 10px;
+  margin-bottom: 10px;
+  background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACwAAAAsCAMAAAApWqozAAAARVBMVEUAAAAzMzMzMzMzMzMzMzMzMzMzMzMzMzMzMzMzMzMzMzMzMzMzMzMzMzP////MzMxZWVny8vLj4+OmpqaMjIyAgIBNTU2nJLiyAAAADXRSTlMAzyAwEEDvv6+PYFDfGXMyHwAAAQBJREFUOMulle0OgjAMRe0GgmABAX3/RzUudJeFgdxwfvFxQm670d2u4/0pzTW1aEDqxh2qbaUJVburlqIbpMyqxUOzPIqMK7qDbKL7u+5y96nrzM3b7n8GIOvcSW3DPPX9NA9JlXBLBe+xWxjfCtBBhPj03Yr+gyDmtkjw6hJeyGJrWcXvmgs7frta2qYGMiCJGi7ITazNjN+NXccqmyDXdjvm5NHe1kF+WnVdIhtD0g9dmPPybO8TeTIxEh5Ma9mjFzkZ/fCUzMTgCxSidcSiEMtNbSRmi1Kbn/mtqB+WGQXUkCHGV8EPRuDkKAM/zPljgj+AcLQ9Tx1toPS3y3wB17hDN7QHhWsAAAAASUVORK5CYII=');
+}
+
+#map .ol-zoom-out {
```
Comment (oterral) [inline less/app.less]: move to map.less

File: less/app.less [hunk 6]
```diff
@@ -39,5 +71,105 @@
...
+  left: auto;
+  width: 44px;
+  background-color: rgba(0,0,0,0);
+}
+
+#map .ol-zoom button {
+  background-color: rgba(0,0,0,0);
+}
+
+#map .ol-zoom-in {
```
Comment (oterral) [inline less/app.less]: move to map.less

File: less/app.less [hunk 7]
```diff
@@ -39,5 +71,105 @@
...
+#map .ol-zoom {
+  position: absolute;
+  top: 98px;
+  right: 10px;
+  left: auto;
+  width: 44px;
+  background-color: rgba(0,0,0,0);
+}
+
+#map .ol-zoom button {
```
Comment (oterral) [inline less/app.less]: move to map.less

File: less/app.less [hunk 8]
```diff
@@ -39,5 +71,105 @@
...
+  position: absolute;
+  right: 0;
+  bottom: 27px;
+  padding: .2em .4em;
+  border-radius: .2em 0 0 0;
+  font-size: 11px;
+  background-color: rgba(255,255,255,0.7);
+}
+
+#attribution {
```
Comment (oterral) [inline less/app.less]: A little bit more less syntax for the `.attribution ` class

File: less/app.less [hunk 9]
```diff
@@ -39,5 +71,105 @@
...
+  line-height: 1.5em;
+}
 
+#footer a {
+  text-decoration: none;
+  font-size: 11px;
+  color: #333;
+}
+
+#footer a:hover {
```
Comment (oterral) [inline less/app.less]: More less syntax

**Comment by oterral** [comment]
A bit more cleaning otherwise looks very good thx for this

**Comment by kandre** [comment]
Very nice !!

File: index.html [hunk 10]
```diff
@@ -26,7 +29,18 @@
       <div id="rightLinks">
         <div id="help">Hilfe</div>
         <div id="print">Drucken</div>
-        <div id="measure">Messen</div>
+        <div id="measure">Messen</span></div>
```
Comment (oterral) [inline index.html]: There is a strange `<span>` here

File: less/search.less [hunk 11]
```diff
@@ -1,6 +1,13 @@
 #search {
   display: inline-block;
   width: 400px;
+  display: inline-block;
+    position: fixed;
+    top: 20px;
+    left: 360px;
+    width: 400px;
+    height: auto;
+    height: 100%;
```
Comment (oterral) [inline less/search.less]: indentations

File: js/map.js [hunk 12]
```diff
@@ -11,6 +11,14 @@ var createMap = function() {
       center: [769500, 155500]
     })
   });
+
+  map.addControl(new ol.control.ScaleLine({target: 'footer'}));
+  map.addControl(new ol.control.Attribution({target: 'attribution', collapsible: false}));
+  
+  $('.ol-viewport .ol-attribution').css('opacity','0');
+  $('.ol-viewport .ol-scale-line').css('opacity','0');
+  $('.ol-zoom-in, .ol-zoom-out').css('color', 'transparent');
```
Comment (oterral) [inline js/map.js]: can you move this 3 line  in map.less ?

File: less/search.less [hunk 13]
```diff
@@ -1,6 +1,13 @@
 #search {
   display: inline-block;
   width: 400px;
+  display: inline-block;
+  position: fixed;
+  top: 20px;
+  left: 360px;
+  width: 400px;
+  height: auto;
+  height: 100%;
```
Comment (oterral) [inline less/search.less]: this one makes the map undraggable

**Comment by oterral** [comment]
@pfanguin Can you remove the css `cursor: move;`  on modal-header (catalog) ?

**Comment by oterral** [comment]
lgtm, can you squash @pfanguin pls?

**Comment by oterral** [comment]
Thx. good work.

**Pull request merged**